src/Estimators/src/FloatingBaseEstimator.cpp

```diff
@@ -0,0 +1,357 @@
+/**
+ * @file FloatingBaseEstimator.cpp
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+
+#include <BipedalLocomotion/Estimators/FloatingBaseEstimator.h>
+
+using namespace BipedalLocomotion::Estimators;
+
+bool FloatingBaseEstimator::initialize(std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
+                                       const iDynTree::Model& model)
+{
+    if (!m_model_comp.setModel(model))
+    {
+        std::cerr << "[FloatingBaseEstimator::initialize] "
+        "The model could not be loaded."
+        << std::endl;
```

Comment:
The text is not well-formatted

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069